modules/apps/object/object-test/src/testFunctional/tests/ObjectPortlet.testcase

```diff
@@ -2566,6 +2566,18 @@ definition {
 		ObjectPortlet.assertEntryNotPresent(entry = "Test text 3");
 
 		ObjectPortlet.assertEntryNotPresent(entry = "Test text 4");
+
+		ObjectPortlet.assertEntryNotPresent(entry = "Test text 5");
+
+		ObjectPortlet.assertEntryNotPresent(entry = "Test text 6");
+
+		ObjectPortlet.assertEntryNotPresent(entry = "Test text 7");
+
+		ObjectPortlet.assertEntryNotPresent(entry = "Test text 8");
+
+		ObjectPortlet.assertEntryNotPresent(entry = "Test text 9");
+
+		ObjectPortlet.assertEntryNotPresent(entry = "Test text 10");
```

Comment:
You can use a loop **(for)** for this case. It's will be easier to read or refactor your code.